examples/oemof_0.2/excel_reader/dispatch.py

```diff
@@ -16,14 +16,17 @@
 
 Installation requirements
 -------------------------
-This example requires the latest version of oemof, matplotlib and xlrd. Install
-by:
+This example requires the latest version of oemof and others.
+Install by:
 
-    pip install 'oemof<0.3,>=0.2'
-    pip install xlrd
-    pip install matplotlib
+    pip3 install 'oemof<0.3,>=0.2'
+    pip3 install xlrd
+    pip3 install matplotlib
+    pip3 install networkx
+    pip3 install pygraphviz
```

Comment:
pygraphviz is difficult to install on some OS (e.g. Windows) therefore I would mark it as optional.